src/components/molecules/SideNavigation/index.tsx

```diff
@@ -44,6 +45,8 @@ const SideNavigation = (props: SideNavigationProps) => {
         [items, currentItemId]
     );
 
+    if (!items.length) return null;
```

Comment:
```suggestion
return !items.length ? null : (
    <div className="side_navigation_container" onClick={handleClick}>
      {SideNavigationItems}
    </div>
  );
```

return을 하나로 합치면 어떤가요? 나중에 컴포넌트가 비대해지면서 if문을 찾지 못해서 힘든일이 있을지도 모를 것 같아요 ㅎㅎ

Reply:
저는 얼리 리턴을 사랑합니다 ㅎㅎ